ddprof-lib/src/main/cpp/safeAccess.h

```diff
@@ -46,7 +47,14 @@ class SafeAccess {
     }
 
     static uintptr_t skipLoad(uintptr_t pc) {
+#ifndef __SANITIZE_ADDRESS__
         if (pc - (uintptr_t)load < 16) {
+#else
+        // asan significantly increases the size of the load function
+        // checking disassembled code can help adjust the value
+        // gdb --batch -ex 'disas _ZN10SafeAccess4loadEPPv' ./elfparser_ut
+        if (pc - (uintptr_t)load < 132) {
```

Comment:
This probably depends on compiler versions.
However for test purposes, I would assume this is enough. WDYT ?

Reply:
I guess I can assert on consecutive function addresses.